tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -2230,7 +2230,33 @@ Status ConvertConv2DHelper(OpConverterParams* params, int group,
     conv_layer = layer;
   }
   nvinfer1::ITensor* output_tensor = conv_layer->getOutput(0);
-
+  // Add an extra padding for Deconv because TRT doesn't accept the
+  // argument output_shape and thus the TRT output shape could be wrong
+  // in case of strides>1.
+  if (is_conv2d_backprop_input) {
+    auto tf_output_shape = backprop_output_size.GetTrtDims();
```

Comment:
I think this should be `backprop_output_size.weights().GetValues()`, but I'm fixing it. Just FYI.

Reply:
hmmm, I am wondering how the test worked?